packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -61,8 +83,9 @@ export class TableInputCSV extends React.PureComponent<Props, State> {
   render() {
     const { width, height } = this.props;
     const { data } = this.state;
+    const styles = getStyles();
     return (
-      <div className="gf-table-input-csv">
+      <div className={styles.gfTableInputCsv}>
```

Comment:
For the naming convention you don't need to be 1-1 with the original class names. In this case I suggest changing the class name to `tableInputCsv`
```suggestion
      <div className={styles.tableInputCsv}>
```